.github/workflows/onPRReviews.yml

```diff
@@ -1,23 +1,120 @@
 name: On review update
 on:
   pull_request:
-    types: [review_requested, review_request_removed]
+    types: [ready_for_review, review_requested, review_request_removed]
   pull_request_review:
     types: [edited, dismissed, submitted]
 jobs:
+  getPRNumber:
+    name: Get Current PR Number
+    runs-on: ubuntu-latest
+    outputs: 
+      issue-number: ${{ steps.condval.outputs.value }}
+    steps:
+      - uses: 3drepo/action-cond@v1
+        id: condval
+        with:
+          cond: ${{ github.event_name == 'pull_request' }}
+          if_true: ${{ github.event.number }}
+          if_false: ${{ github.event.pull_request.number }}
+
+  getIssueNumber:
+    name: Get Current Issue Number
+    needs: [ getPRNumber ]
+    runs-on: ubuntu-latest
+    outputs: 
+      issue-number: ${{ steps.commitMsgParser.outputs.issue-number }}
+      content-id: ${{ steps.commitMsgParser.outputs.content-id }}
+    steps:
+      - name: Extract Issue Number from PR
+        uses: 3drepo/extract-issue-number-from-PR@v1
+        with:
+          pr: ${{ needs.getPRNumber.outputs.issue-number }}
+          base: staging
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+        id: commitMsgParser
+
   checkMergeReady:
     name: Ensure all reviewers have approved the PR
     runs-on: ubuntu-latest
+    needs: [ getIssueNumber ]
+    outputs: 
+      approved: ${{ steps.checkReviews.outputs.approved }}
     steps:
-      - uses: haya14busa/action-cond@v1
+      - uses: 3drepo/action-cond@v1
         id: condval
         with:
           cond: ${{ github.event_name == 'pull_request' }}
           if_true: ${{ github.event.number }}
           if_false: ${{ github.event.pull_request.number }}
       - name: Check reviews
-        uses: carmenfan/all-reviewers-approved@v1.0
+        id: checkReviews
+        uses: 3drepo/all-reviewers-approved@v1
         with:
           pr: ${{ steps.condval.outputs.value }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update status
```

Comment:
wrong place for this. just because all the reviewers approved doesn't necessarily mean the issue is done

The issue should only be considered done when it's merged - the best place to do this is probably in https://github.com/3drepo/3drepo.io/blob/master/.github/workflows/onStagingCommits.yml